### DISCUSSION TITLE
QFieldCloud planned maintanance on 10th August

### DISCUSSION THREAD

**Discussion by suricactus** [opengisch/QField#3181] [Announcements]
For a few hours QFieldCloud will not be available as a service.

**Comment by gabgarcies** [comment]
Hi Suricatus,

Is this why I do not see the majority of my projects on the website?

I see something new in the projects remaining:

![image](https://user-images.githubusercontent.com/80522138/183844615-c673bdf6-b327-4ea7-b370-5df593f6314a.png)

**Reply by Alekssssanderr** [reply]
I have this same problem. 70% of my project dissapear from cloud. Can you check this? Or maybe now theres a limit of project number?

**Reply by Notramus** [reply]
I have the same problem as @Alekssssanderr , but additionally nearly all my collaborators were kicked out of the still remaining projects.

**Comment by DumitruD87** [comment]
I have a problem. My entire project was deleted from the cloud. What should I do? Should I wait after the maintenance?

**Comment by suricactus** [comment]
@gabgarcies  @Alekssssanderr @Notramus @DumitruD87 we are looking into your issues.

Your projects are most probably not deleted, but unaccessible, since we enforced some new account permission rules that are close to what is going to be when QFieldCloud goes out of BETA. Please contact the project owner of the "missing" projects, they should see them and fix the issue.

Something you should be aware of is that you are allowed of maximum of 1 coollaborator per private project. Please transfer your projects to an organization to collaborate with more users.

**Reply by DumitruD87** [reply]
![image](https://user-images.githubusercontent.com/110976200/183884776-84f629e6-738b-4323-8a9a-72c2cc5a1bb1.png)
I am the project owner of the "missing" projects. But i can't see all my project. One is missing.

**Reply by suricactus** [reply]
thanks for the screenshot, I will keep you posted. Your project is intact and it is not showing because you have more collaborators than allowed. Obviously not showing the project is not ok and we are working on it.

**Reply by DumitruD87** [reply]
OK. Good. Can all collaborators be deleted? To stay only me?

**Reply by DumitruD87** [reply]
Thank you very much!

**Reply by Alekssssanderr** [reply]
@suricactus I'm the project owner and I can't see missing project in cloud. What should I do?

**Reply by Notramus** [reply]
@suricatus I am as well the project owner. Thanks for your help. I will be waiting until the project appears.

**Comment by gabgarcies** [comment]
Hi,

OK, I might understand why the projects have disappeared.

The missing projects are private projects (out of any organization) and they have more than 2 collaborators.

In my case, I am the owner of all the missing projects (more than 10) and they usually have 5 collaborators.

I have no problem at putting them into an organization and starting a billing plan. Actually, we do have an organization created in QfieldCloud (FUEIB_Eivissa), but the projects had not been created in it.

Now, I am not able to insert the missing projects into the organization (as I am not seeing them anymore on the website).

My question is:

***Would that be possible for you to make this change for me: putting my private projects (user gabgarcies) into the organization I mentioned before (FUEIB_Eivissa)?***

If not, what alternatives do I have? The biggest problem now is that my fieldworkers continue collecting data, as they had already downloaded the projects on their devices. We do not know what will happen once they push changes.

Thank you so much,

Gabriel Garcies

**Reply by suricactus** [reply]
Hey, @gabgarcies you are following the conditions correctly. The interface willl suggest you to transfer them to an organization. This is already implemented. The issue is that the projects are hidden you cannot see this page. It's something we identified and working on it.

Your field collectors should be able to push the project once it has the projects have valid number of users according to your plan (e.g. organizations are limitless).

**Reply by gabgarcies** [reply]
OK, I will do the transfer to an organization as soon as I recover the projects unseen.

In the meanwhile, can my fieldworkers continue working locally or what do you recommend me?

**Reply by gabgarcies** [reply]
Everything works fine and I have been able to move the projects to an organization.

Thank you!

**Comment by suricactus** [comment]
@gabgarcies @Alekssssanderr @Notramus @DumitruD87 @juwizeye @TheJoseph24 I hope now you can enjoy the new features of QFieldCloud without an issue.
